lisa/notebook.py

```diff
@@ -376,7 +376,7 @@ def make_figure(width, height, nrows, ncols, interactive=None, **kwargs):
     return (figure, axes)
 
 
-def plot_signal(series, name=None, interpolation=None, add_markers=True):
+def plot_signal(series, name=None, interpolation=None, add_markers=True, vdims=None):
```

Comment:
the docstring needs to be updated. Does it make sense to allow for multiple vdims even though only a single Series can be plotted by this function ?